netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -122,6 +129,7 @@ public int getPort() {
     return ((InetSocketAddress) localAddr).getPort();
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Instead of disabling the checking for the entire (large) method, make a temporary variable to apply the annotation to:
```java
@SuppressWarnings("unchecked")
ChannelOption<Object> key = (ChannelOption<Object>) entry.getKey();
b.childOption(key, entry.getValue());
```

Reply:
OK.